### PR TITLE
Environment_oM: Updates to gains

### LINKED ISSUE #516: Environment_oM: Update LightingGain Properties
#### Description:
Add Sensible Gain property to LightingGain:
``` C#
public double SensibleGain { get; set; } = 0.0;
```
FYI @FraserGreenroyd

### LINKED ISSUE #517: Environment_oM: Update SpaceType Enum
#### Description:
Space Type list Enum to be adjusted

### PR DESCRIPTION
**Pull request by FraserGreenroyd**
### Issues addressed by this PR
Fixes #516 
Fixes #517 

### Test files
N/A

#### Changed
 - Changed Gains properties to better match what engineers would expect to see

### Additional comments
Please do not merge until dependent PRs are also ready.

### CODE REVIEW

File: Environment_oM/Gains/People.cs [hunk 1]
```diff
@@ -30,10 +30,12 @@
...
+    public class People : BHoMObject, IGain
     {
+        public double Sensible { get; set; } = 0.0;
+        public double Latent { get; set; } = 0.0;
         public Profile Profile { get; set; } = new Profile();
-        public GainUnit Unit { get; set; } = GainUnit.Undefined;
+
         public double RadiantFraction { get; set; } = 0.0;
         public double ViewCoefficient { get; set; } = 0.0;
         public double Value { get; set; } = 0.0;
```
Comment (al-fisher) [inline Environment_oM/Gains/People.cs]: ``` public double Value { get; set; } = 0.0;```  - needs to be removed

Comment (FraserGreenroyd) [reply Environment_oM/Gains/People.cs]: Done!

**Comment by FraserGreenroyd** [comment]
@al-fisher changes made and pushed to PR :smile:

**Comment by al-fisher** [comment]
Thanks for that @FraserGreenroyd - now LGTM!

**Pull request merged**